### ISSUE TITLE
beancount uses BASE/QUOTE inconsistently (regarding whitespace)

### ISSUE THREAD

**Issue by blais** [beancount/beancount#252]
**[Original report](https://bitbucket.org/blais/beancount/issue/252) by Martin Michlmayr (Bitbucket: [tbm13](https://bitbucket.org/tbm13), GitHub: [tbm](https://github.com/tbm)).**

----------------------------------------

`bean-report foo commodities` shows me:

```
USD / UR                 
```

but when I use this pair as printed (with spaces) I get an error:

```
bean-report foo prices -c "USD / UR"
bean-report: error: Invalid commodity pair "USD / UR"; must be in BASE/QUOTE format
```

When I remove the spaces it works: "USD/UR"

Either `commodities` shouldn't print spaces or spaces should be accepted.

**Comment by blais** [comment]
**Original comment by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

Merged in tbm13/beancount/bug252 (pull request #60)

Fixed #252: allow whitespace in BASE/QUOTE

**Comment by blais** [comment]
**Original comment by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

Fixed #252: allow whitespace in BASE/QUOTE

**Comment by blais** [comment]
**Original comment by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

Merged in tbm13/beancount/bug252 (pull request #60)

Fixed #252: allow whitespace in BASE/QUOTE
